sql/ganneffserv-pgsql.sql

```diff
@@ -8,4 +8,14 @@ CREATE TABLE ganneffserv (
   kills   INTEGER NOT NULL DEFAULT 0,
   monitor_only BOOLEAN NOT NULL DEFAULT 'False'
 );
+
+DROP TABLE IF EXISTS ganneffprotect CASCADE;
+CREATE TABLE ganneffprotect (
+  id      SERIAL PRIMARY KEY,
+  setter  INTEGER REFERENCES account(id) ON DELETE SET NULL,
+  time    INTEGER NOT NULL,
+  pattern VARCHAR(255) NOT NULL,
+  reason  VARCHAR(255) NOT NULL
+)
+
 CREATE UNIQUE INDEX ganneffserv_channel_idx ON ganneffserv (irc_lower(channel));
```

Comment:
```suggestion
);
CREATE UNIQUE INDEX ganneffserv_channel_idx ON ganneffserv (irc_lower(channel));

DROP TABLE IF EXISTS ganneffprotect CASCADE;
CREATE TABLE ganneffprotect (
  id      SERIAL PRIMARY KEY,
  setter  INTEGER REFERENCES account(id) ON DELETE SET NULL,
  time    INTEGER NOT NULL,
  pattern VARCHAR(255) NOT NULL,
  reason  VARCHAR(255) NOT NULL
)
CREATE UNIQUE INDEX ganneffservprotect_pattern_idx ON ganneffservprotect(irc_lower(pattern));
```

(Put the table creation below all the existing schema, and add a unique index, which is how Postgres enforces unique constraints)